src/misc/throwables.ts

```diff
@@ -14,6 +14,9 @@ const THROW_SOUND = Sound.ENTITY_SNOWBALL_THROW;
 const HIT_SOUND = Sound.BLOCK_STONE_HIT;
 const ZERO_VECTOR = new Vector();
 
+const THROW_COOLDOWN = 80; // ms
```

Comment:
This will wait either 1 or 2 ticks. Consider using ticks instead of milliseconds for delays.